payjoin/src/send/v1.rs

```diff
@@ -51,7 +51,8 @@ pub struct SenderBuilder<'a> {
 /// We only need to add the weight of the txid: 32, index: 4 and sequence: 4 as rust_bitcoin
 /// already accounts for the scriptsig length when calculating InputWeightPrediction
 /// <https://docs.rs/bitcoin/latest/src/bitcoin/blockdata/transaction.rs.html#1621>
-const NON_WITNESS_INPUT_WEIGHT: bitcoin::Weight = Weight::from_non_witness_data_size(32 + 4 + 4);
+pub(crate) const NON_WITNESS_INPUT_WEIGHT: bitcoin::Weight =
```

Comment:
Same thing here. Lets revert this. If we are going to change it lets do it in a seperate commit